bokehjs/src/coffee/models/layouts/layout_dom.coffee

```diff
@@ -38,9 +38,14 @@ class LayoutDOMView extends BokehView
       _.delay(resize, 100)
       _.delay(resize, 200)
 
+    if @model.responsive is 'fixed'
```

Comment:
I already know you would rather not, because I examined your recent contributions. Using two operators for one thing is one too many. We default to `==` and that's what you should use. When you use `is`, you actually increase confusion.